core/installation/steps/database_configuration.php

```diff
@@ -37,7 +37,12 @@
 
         try {
             // This throws a PDOException if the connection fails
-            DB::getCustomInstance($db_address, $db_name, $db_username, $db_password, $db_port, $force_charset=$db_charset);
+            $db = DB::getCustomInstance($db_address, $db_name, $db_username, $db_password, $db_port, $force_charset=$db_charset);
```

Comment:
`$force_charset=$db_charset`
Neither variable here exists, and the `=` is clearly not the correct syntax here.
What is intended behavior?

I think we should always force charset of `utf8mb4` and this param can probably be removed.